src/undo_log/undo_log.h

```diff
@@ -65,13 +67,25 @@ typedef struct {
 	SIValue orig_value;           // attribute original value
 } UndoUpdateOp;
 
+typedef struct {
```

Comment:
you can leave one struct with name UndoLabelsOp and the UndoOpType will just tell you if this remove or set